src/Dotnet.ProjInfo.Workspace/MSbuildInfo.fs

```diff
@@ -5,25 +5,8 @@ module MSBuildInfo =
   open System
   open System.IO
 
-  let private programFilesX86 () =
-      let environVar v = Environment.GetEnvironmentVariable v
-
-      let wow64 = environVar "PROCESSOR_ARCHITEW6432"
-      let globalArch = environVar "PROCESSOR_ARCHITECTURE"
-      match wow64, globalArch with
-      | "AMD64", "AMD64"
-      | null, "AMD64"
-      | "x86", "AMD64" -> environVar "ProgramFiles(x86)"
-      | _ -> environVar "ProgramFiles"
```

Comment:
So it seems that the old implementation could return `ProgramFiles` rather than `ProgramFiles(x86)` in some cases, and as far as I understand the new one will always return only x86 version.

Reply:
Perhaps, I am not sure, but the existing logic might fail when the system drive is not `C`. In any case, `Environment.GetFolderPath` is the recommended way to get a system folder path.